app/services/LoginService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\LoginModel;
+use App\Models\Repositories\LoginRepository;
+
+class LoginService
+{
+    public function __construct(
+        private readonly LoginRepository $repository
+    ) {
+    }
+
+    public function registerUser(LoginModel $validationFields)
```

Comment:
Should: The **Login**Service shouldn't contain **registerUser** method. **S**OLID